src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -5,22 +5,20 @@
 
 public class SalaryInfo {
 
-    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
 
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
         StringBuilder salaryReportBuilder = new StringBuilder();
         int[] salary = new int[names.length];
-        LocalDate dateFromFormated = LocalDate.parse(dateFrom, formatter);
-        LocalDate dateToFormated = LocalDate.parse(dateTo, formatter);
```

Comment:
You should keep these variables. Now when you are calling your new method (`isDataWithinRange`) you are unnecessary creating 2 new variables on each method call. Rework your new method a bit, so that it accepts as parameters already formated `dateFrom` and `dateTo`.